### ISSUE TITLE
save_PNG format

### ISSUE THREAD

**Issue by hnyoBus** [proplot-dev/proplot#379]
```python
import proplot as pplt
import numpy as np
fig, ax= pplt.subplots(nrows=4, ncols=1, space=0, refwidth="5cm", refaspect=(5, 1))

x = np.linspace(0,2*np.pi,50)
y = np.sin(x)

for i in range(4):
    ax[i].plot(x,y, label = "sin")

ax.format(
grid = False,
yticks=[],

)
ax[0].legend(fontsize = 5, title = "my sin", titlefontsize = 5) # titlefontsize is OK, while fontsize = 10 seem not work ??
fig.save("ppro_sin.png", dpi=300, transparent =True) # when save with transparent , all the xticks would apear?
```

**Comment by lukelbd** [comment]
This was fixed in #331, but unfortunately is not in any new releases. For now you can solve it by installing the dev version with: `pip install git+https://github.com/proplot-dev/proplot.git`.

**Comment by hnyoBus** [comment]
已收到，谢谢你！
